### ISSUE TITLE
What is the training/test/validation image ids in DFC 2019?

### ISSUE THREAD

**Issue by lauraset** [CVEO/S3Net#1]
@1128yqy , thank you for your great work. I want to know that the specific training/test/validation images in DFC 2019. Can you share your datalist? Thank you in advance.

**Comment by 1128yqy** [comment]
Okey, here is how I divide my dataset: the series 314, 359, 412, 416, 427, 467, 474, 505, and 559 from the JAX dataset are allocated to the test set. Meanwhile, OMA 382, 389, 391, 376, and JAX 264, 269, 276, 280 are set as the validation set, with the remaining data used for the training set.

**Comment by 1128yqy** [comment]
> @1128yqy , thank you for your great work. I want to know that the specific training/test/validation images in DFC 2019. Can you share your datalist? Thank you in advance.

Okey, here is how I divide my dataset: the series 314, 359, 412, 416, 427, 467, 474, 505, and 559 from the JAX dataset are allocated to the test set. Meanwhile, OMA 382, 389, 391, 376, and JAX 264, 269, 276, 280 are set as the validation set, with the remaining data used for the training set.

**Comment by lauraset** [comment]
@1128yqy . Thank you very much, Qingyuan. 
By the way, I want to know that for other methods (PSMNet, GwcNet, GANet, CFNet, and S2Net), you directly used the results from the paper in S2Net?  Have you ever reproduced these methods? For example, S2NET. They did not release the code yet.

**Comment by 1128yqy** [comment]
> @1128yqy . Thank you very much, Qingyuan. By the way, I want to know that for other methods (PSMNet, GwcNet, GANet, CFNet, and S2Net), you directly used the results from the paper in S2Net? Have you ever reproduced these methods? For example, S2NET. They did not release the code yet.

Yes, the results of models like PSMNet and GwcNet have been validated in the S2Net paper, and that's where my results originate. However, I regret that, even though I possess the source code for S2Net, the owners of S2Net do not have plans to open-source it.

**Comment by lauraset** [comment]
@1128yqy . I understand. Thank you for your reply and the open release of codes.
